src/api/messages.ts

```diff
@@ -0,0 +1,23 @@
+import { HTTPError } from 'ky';
```

Comment:
I am getting error with my account on most of the events

![CleanShot 2024-05-17 at 13 03 12](https://github.com/GLObus303/sportujspolu/assets/12488648/d0186f11-4cad-43e6-be00-0cce7c7e8460)